smack-extensions/src/test/java/org/jivesoftware/smackx/VCardUnitTest.java

```diff
@@ -35,40 +37,222 @@ public void initSmackTestSuite() {
     }
 
     @Test
+    public void testParseFullVCardIQStanza() throws Throwable {
+
+        // @formatter:off
+        final String request =
+        "<iq id='v1' to='user@igniterealtime.org/mobile' type='result'>"
```

Comment:
One thing I'm not clear about is whether this XEP is case sensitive. Currently all of the tags are matched only in their upper case form, which is how they appear in the spec. Should I refactor to accept them in a case insensitive way?